### PR TITLE
Repository setup

### LINKED ISSUE #4: Setting up repository settings and merge rules and basic templates
Setting up a basic repository:
- [x] Repository Settings
- [x] Protected branches
- [x] Some basic templates regarding issues / prs for consistency
- [x] Contributing guide
- [x] README
    - [x] Links to the major parties involved
        - [x] USDX
        - [x] World Party
        - [x] Vocaluxe
        - [x] Performous
        - [x] MyLittleKaraoke
        - [x] USDB website
        - [x] ultrastar-es website
        - [x] Yass (reloaded?)
        - [x] composer
        - [x] Karedi
        - [x] UltraStar Creator
        - [x] UltraStar Manager
        - [x] USDB-Syncer

### PR DESCRIPTION
**Pull request by Baklap4**
### What does this PR do?

* Sets up the repository as described in #4 
    * EditorConfig added so we maintain the same formatting in every file.
    * Added links to involved parties
    * Added some Github templates

### Closes Issue(s)

Closes #4 

### Motivation

A repository about standardization and formalization should in itself also be structured as otherwise it might defeat the purpose already by not giving a good example.

### More

- [x] Added/updated documentation

### Additional Notes

Please check if all the links are working

### CODE REVIEW

**Review by bohning — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,3 +13,31 @@ The endgoal for this project would be to:
 * [ ] Bring the different communities on par with eachother to increase the quality of the produced textfiles
 * [ ] Versionise the format so we can improve and add features to it
 * [ ] Give each Tag or NoteType a definition and how it should be working
+
+## Involved parties
+Since the text format was introduced a little bit more than a decade ago alot of projects got to know and support it.
+We'd like to give these projects a little bit of appreciation by linking to their respective GitHub page or website.
+
+### Karaoke software
```
Comment (Baklap4) [inline README.md:21]: Missing UltraStar Play

**Review by basisbit — CHANGES_REQUESTED** [review]
Please add UltraStar Play / Melody Mania to the list.

File: README.md (hunk 1 above)
Comment (Baklap4) [reply README.md:21]: Added

**Comment by Baklap4** [comment]
> Please add UltraStar Play / Melody Mania to the list.

@basisbit Updated the branch to include UltrStar Play and Melody Mania

**Review by basisbit — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -13,3 +13,31 @@ The endgoal for this project would be to:
...
+* [Yass Reloaded](https://github.com/DoubleDee73/Yass)
+* [Composer](https://github.com/performous/composer)
+* [Karedi](https://github.com/Nianna/Karedi)
+* [UltraStar Creator](https://github.com/UltraStar-Deluxe/UltraStar-Creator)
+
+### UltraStar Textfiles hosting
+* [USDB](https://usdb.animux.de)
+* [UltraStar-ES](http://ultrastar-es.org/)
+
+### Other Software
```
[the remark this answers is not in the document]
Comment (Baklap4) [reply README.md]: Updated to make it say "Management software" instead of "Other Software"

And moved UltraSinger to Creator software section. Thanks!

**Review by rakuri255 — APPROVED** [review]

**Review by rakuri255 — APPROVED** [review]

**Pull request merged**